azure-common/azure/profiles/__init__.py

```diff
@@ -149,6 +149,53 @@ class KnownProfiles(Enum):
         },
         "2018-03-01-hybrid"
     )
+    v2019_03_01_hybrid = ProfileDefinition(
+        {
+            "azure.keyvault.KeyVaultClient": {
+                None: "2016-10-01"
+            },
+            "azure.multiapi.storage": {
+                None: "2017-11-09"
+            },
+            "azure.multiapi.cosmosdb": {
+                None: "2017-04-17"
+            },
+            "azure.mgmt.authorization.AuthorizationManagementClient": {
+                None: "2015-07-01"
+            },
+            "azure.mgmt.compute.ComputeManagementClient": {
+                None: "2017-12-01"
+            },
+            "azure.mgmt.keyvault.KeyVaultManagementClient":{
+                None: "2016-10-01"
+            },
+            "azure.mgmt.network.NetworkManagementClient": {
+                None: "2017-10-01"
+            },
+            "azure.mgmt.storage.StorageManagementClient": {
+                None: "2017-10-01"
+            },
+            "azure.mgmt.resource.policy.PolicyClient": {
+                None: "2016-12-01"
+            },
+            "azure.mgmt.resource.locks.ManagementLockClient": {
+                None: "2016-09-01"
+            },
+            "azure.mgmt.resource.links.ManagementLinkClient": {
+                None: "2016-09-01"
+            },
+            "azure.mgmt.resource.resources.ResourceManagementClient": {
+                None: "2018-05-01"
+            },
+            "azure.mgmt.resource.subscriptions.SubscriptionClient": {
+                None: "2016-06-01"
+            },
+            "azure.mgmt.dns": {
+                None: "2016-04-01"
+            }
+        },
+        "2019-03-01-hybrid"
+    )
```

Comment:
We should include the Insights (Azure Monitor) here.

Reply:
We don't have azure monitor - multi api package yet since Azure Stack supports the latest.
From Laurent:

> For Monitor (formerly Insights), the latest API version available on the SDK and public Azure are exactly the one you have in your profile, then multi-api package doesn’t seem necessary to me. If Monitor updates their API (which is not likely, I release rarely monitor currently), we can update to multi-api at that time without breaking the CLI. So I would suggest to keep it stupid and simple and do nothing for now.